.circleci/real_config.yml

```diff
@@ -98,6 +101,10 @@ release-dryrun: &release-dryrun
       # Tags like: v3.1.3+dryrun
       - /v\d+\.\d+\.\d+\+dryrun/
 
+release-upgrade: &release-upgrade
```

Comment:
@mackrorysd it was easier to just keep this in the circleCI file (instead of a newline delimited file), so what are the initial branches I should replace here for the upcoming release?